code/game/mecha/mecha_construction_paths.dm

```diff
@@ -1591,3 +1591,216 @@
+					holder.icon_state = "hermes5"
+			if(7)
+				if(diff==FORWARD)
+					user.visible_message("[user] secures the peripherals control module.", "You secure the peripherals control module.")
+					holder.icon_state = "hermes8"
+				else
+					user.visible_message("[user] removes the peripherals control module from [holder].", "You remove the peripherals control module from [holder].")
+					new /obj/item/weapon/circuitboard/mecha/hermes/peripherals(get_turf(holder))
+					holder.icon_state = "hermes6"
+			if(6)
+				if(diff==FORWARD)
+					user.visible_message("[user] installs internal armor layer to [holder].", "You install internal armor layer to [holder].")
+					holder.icon_state = "hermes9"
+				else
+					user.visible_message("[user] unfastens the peripherals control module.", "You unfasten the peripherals control module.")
+					holder.icon_state = "hermes7"
+			if(5)
+				if(diff==FORWARD)
+					user.visible_message("[user] secures internal armor layer.", "You secure internal armor layer.")
+					holder.icon_state = "hermes10"
+				else
+					user.visible_message("[user] pries internal armor layer from [holder].", "You prie internal armor layer from [holder].")
+					var/obj/item/stack/material/steel/MS = new /obj/item/stack/material/steel(get_turf(holder))
+					MS.amount = 5
+					holder.icon_state = "hermes8"
+			if(4)
+				if(diff==FORWARD)
+					user.visible_message("[user] welds internal armor layer to [holder].", "You weld the internal armor layer to [holder].")
+					holder.icon_state = "hermes11"
+				else
+					user.visible_message("[user] unfastens the internal armor layer.", "You unfasten the internal armor layer.")
+					holder.icon_state = "hermes9"
+			if(3)
+				if(diff==FORWARD)
+					user.visible_message("[user] installs [used_atom] layer to [holder].", "You install external reinforced armor layer to [holder].")
+
+					holder.icon_state = "hermes12"
+				else
+					user.visible_message("[user] cuts internal armor layer from [holder].", "You cut the internal armor layer from [holder].")
+					holder.icon_state = "hermes10"
+			if(2)
+				if(diff==FORWARD)
+					user.visible_message("[user] secures external armor layer.", "You secure external reinforced armor layer.")
+					holder.icon_state = "hermes13"
+				else
+					var/obj/item/stack/material/plasteel/MS = new /obj/item/stack/material/plasteel(get_turf(holder))
+					MS.amount = 5
+					user.visible_message("[user] pries [MS] from [holder].", "You prie [MS] from [holder].")
+					holder.icon_state = "hermes11"
+			if(1)
+				if(diff==FORWARD)
+					user.visible_message("[user] welds external armor layer to [holder].", "You weld external armor layer to [holder].")
+					holder.icon_state = "hermes14"
+				else
+					user.visible_message("[user] unfastens the external armor layer.", "You unfasten the external armor layer.")
+					holder.icon_state = "hermes12"
+		return 1
+
+	spawn_result()
+		..()
+		feedback_inc("mecha_hermes_created",1)

```

Comment:
@Arrow768 do we even have feedback database?

Reply:
Yes, but atm we dont really have any stats systems that can be viewed by the public.